image/templates/helm/shared/templates/02-scanner-v4-07-indexer-deployment.yaml

```diff
@@ -82,17 +82,18 @@ spec:
             drop: ["NET_RAW"]
           runAsNonRoot: true
           runAsUser: 65534
-# TODO(mclamei): Enable when the grpc healthcheck in indexer works.
-#
-# {{ if semverCompare ">= 1.24.0" ._rox._apiServer.version }}
-#         readinessProbe:
-#           grpc:
-#             port: 8443
-#           timeoutSeconds: 10
-#           periodSeconds: 10
-#           failureThreshold: 6
-#           successThreshold: 1
-# {{- end }}
+        livenessProbe:
+          httpGet:
+            scheme: HTTPS
```

Comment:
Just want to confirm this worked, right? no cert validation errors or anything?

Reply:
See "testing" in the description.